src/Policy/AbstractAccessPolicy.php

```diff
@@ -73,12 +73,10 @@ public static function listActions()
     {
         static $actions = [];
```

Comment:
This doesn't need to be static any more.

```suggestion
        $actions = [];
```